Simplenote/src/androidTest/java/com/automattic/simplenote/uipages/LoginPage.kt

```diff
@@ -0,0 +1,12 @@
+package com.automattic.simplenote.uipages
+
+import androidx.test.uiautomator.UiObject
+
+const val LOGINPAGE_LOGINBUTTON: String = "com.automattic.simplenote.debug:id/button"
+const val FORGOT_PASSWORD_LINK: String = "com.automattic.simplenote.debug:id/text_footer"
+const val PASSWORD_HIDE_IMAGE: String = "com.automattic.simplenote.debug:id/text_input_end_icon"
+
+class LoginPage: BasePage() {
+    val loginPageLoginButton: UiObject = getUIDeviceInstance().findObject(getUISelectorInstance().resourceId(LOGINPAGE_LOGINBUTTON))
```

Comment:
Since we are in the context of `LoginPage`, we already know that we are on the login page, so something like `LoginPage().loginPageLoginButton` looks a bit redundant. Because of this, it could be just something like:

```suggestion
    val loginButton: UiObject = getUIDeviceInstance().findObject(getUISelectorInstance().resourceId(LOGIN_BUTTON))
```

Reply:
agree, makes sense.